dashboard/client/dashboard.js

```diff
@@ -5,8 +5,74 @@ Template.dashboard.onCreated(function () {
   // Get reference to template instance
   const templateInstance = this;
 
-  // Create reactive variable for Elasticsearch host
+  // Create reactive variable for Elasticsearch host & data
   templateInstance.elasticsearchHost = new ReactiveVar();
+  templateInstance.elasticsearchData = new ReactiveVar();
+
+  const queryParams = {
+    size: 0,
+    body: {
+      query: {
+        filtered: {
+          query: {
+            bool: {
+              should: [
+                {
+                  wildcard: {
+                    request_path: {
+                      // Add '*' to partially match the url
+                      value: '/*',
+                    },
+                  },
+                },
+              ],
+            },
+          },
+        },
+      },
+      aggs: {
+        // Create data range with interval
+        requests_over_time: {
+          date_histogram: {
+            field: 'request_at',
+            interval: 'month',
+          },
+          aggs: {
+            // Get average value of response time for each range
+            avg_response_time: {
+              avg: {
```

Comment:
I can let go of the idea to use percentiles, since they are expensive to compute. However, lets explore the concept of average a bit more.

When you say 'average' what do you mean?